pipelines/datasets/br_inmet_bdmep/utils.py

```
@@ -232,6 +237,99 @@ def get_clima_info(file: str) -> pd.DataFrame:
+        "altitude": float(df_estacao.loc[6, "value"].replace(",", "."))
+        if df_estacao.loc[6, "value"] != ""
+        else "",
+        "data_fundacao": "",
+    }
+
+    if "/" in df_estacao.loc[7, "value"]:
+        try:
+            dados_estacao["data_fundacao"] = datetime.strptime(
+                str(df_estacao.loc[7, "value"]), "%Y/%m/%d"
+            )
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)